extensions/amp-geo/amp-geo.md

```diff
@@ -167,12 +167,17 @@ example below.
 </amp-geo>
 ```
 
-#### U.S. Virginia Detection
+#### U.S. Virginia, Colorado, Connecticut Detection
```

Comment:
Are there still good reasons this section is separate from that of California above?

Reply:
i dont believe so, the only difference is we added a "preset-us-ca" which doesn't make sense to me since it seems to be emulating `preset-eea` for European countries grouping. But unless we support city(?) detection (or state region?) for California the only thing we can match for `preset-us-ca` is California (us-ca). I think it should have been `preset-us` instead of `preset-us-ca`.

I was wanting to clean it up, but don't want to break the `preset-us-ca` feature for the publishers that already use it.